core/Cargo.toml

```diff
@@ -12,6 +12,7 @@ categories = ["network-programming", "asynchronous"]
 [dependencies]
 asn1_der = "0.6.1"
 bs58 = "0.3.0"
+bytes = "0.6.0"
```

Comment:
To avoid problems, I think it best to stick to `0.5` in this PR and leave the `0.6` update to https://github.com/libp2p/rust-libp2p/pull/1810.

Reply:
As this PR needs changes on rust-multihash which aren't merged yet, it will take some time on that side. I'll take it into account it case that merging moves faster than the dependency upgrade.